apps/src/aiTutor/views/AITutorFooter.tsx

```diff
@@ -0,0 +1,150 @@
+import React, {useState, useCallback} from 'react';
+import style from './ai-tutor.module.scss';
+import Button from '@cdo/apps/templates/Button';
+import {useAppDispatch, useAppSelector} from '@cdo/apps/util/reduxHooks';
+import {askAITutor} from '@cdo/apps/aiTutor/redux/aiTutorRedux';
+import analyticsReporter from '@cdo/apps/lib/util/AnalyticsReporter';
+import {EVENTS} from '@cdo/apps/lib/util/AnalyticsConstants';
+
+import {
+  AITutorTypes as ActionType,
+  AITutorTypesValue,
+} from '@cdo/apps/aiTutor/types';
+
+const QuickActions = {
+  [ActionType.COMPILATION]: "Why doesn't my code compile?",
+  [ActionType.VALIDATION]: "Why aren't my tests passing?",
+};
+
+interface AITutorFooterProps {
+  renderAITutor: boolean;
+}
+
+const AITutorFooter: React.FC<AITutorFooterProps> = ({renderAITutor}) => {
+  const [userMessage, setUserMessage] = useState<string>('');
+
+  const isWaitingForChatResponse = useAppSelector(
+    state => state.aiTutor.isWaitingForChatResponse
+  );
+
+  const level = useAppSelector(state => state.aiTutor.level);
```

Comment:
Most of this logic isn't new, and is borrowed from a previous component called `userChatMessageEditor`: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/code-studio/components/aiTutor/userChatMessageEditor.tsx

Here's the diff...
<img width="1211" alt="Screenshot 2024-04-10 at 11 20 17 PM" src="https://github.com/code-dot-org/code-dot-org/assets/26844240/d9f87a6a-4163-424d-a758-b432c6633399">
<img width="1217" alt="Screenshot 2024-04-10 at 11 20 30 PM" src="https://github.com/code-dot-org/code-dot-org/assets/26844240/b5c939da-112b-4c66-9127-09663c77c168">
<img width="1211" alt="Screenshot 2024-04-10 at 11 20 45 PM" src="https://github.com/code-dot-org/code-dot-org/assets/26844240/4e043e5b-8ae3-43f2-bfdf-eaefbfe34d8e">
<img width="1211" alt="Screenshot 2024-04-10 at 11 21 00 PM" src="https://github.com/code-dot-org/code-dot-org/assets/26844240/0dfa6c74-73bb-4bc2-a9f7-41c442aa8570">
<img width="1219" alt="Screenshot 2024-04-10 at 11 21 14 PM" src="https://github.com/code-dot-org/code-dot-org/assets/26844240/1ae71ceb-cbbc-4bba-9448-416bd5a6150a">
<img width="1219" alt="Screenshot 2024-04-10 at 11 21 28 PM" src="https://github.com/code-dot-org/code-dot-org/assets/26844240/21d669b1-29cc-4ea1-9317-fbb1ca98dd16">